src/main/resources/assets/appliedenergistics2/lang/en_US.lang

```diff
@@ -202,6 +202,7 @@ gui.appliedenergistics2.Substitute=Using Substitutions:
 gui.appliedenergistics2.BeSubstitute=Can be a substitution:
 gui.appliedenergistics2.Yes=Yes
 gui.appliedenergistics2.No=No
+gui.appliedenergistics2.AuthoredBy=Authored by: %s
```

Comment:
Would suggest: "Encoder:", "Encoded by:", etc as Authored By makes it sound like a git commit

Reply:
Encoded by sounds better